code/modules/atmospherics/machinery/components/unary_devices/passive_vent.dm

```diff
@@ -22,7 +22,11 @@
 /obj/machinery/atmospherics/components/unary/passive_vent/process_atmos()
 	..()
```

Comment:
This is somewhat out of scope,  but do you mind yeeting this? it doesn't do anything

Reply:
sure

Reply:
i've removed it from the scrubbers too while i was at it